README.md

```diff
@@ -35,9 +35,11 @@ Short description of the resource. It’s recommended to copy and paste the best
 - Topics: Topic 1, Topic 2, Topic 3
 ```
 
+---
+
```

Comment:
This needs to be removed.

Reply:
@thetaPC I added this on purpose to separate the Contributing section from when the Resources start. Is it confusing to have the line here?

Reply:
You have a great point and I like the reasoning. Let's keep it in.